mirascope/integrations/langfuse/_utils.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, Callable
+
+from ...core.base import BaseCallResponse
+from ...core.base._stream import BaseStream
+from ...core.base.metadata import Metadata
+
+
+def get_call_response_observation(result: BaseCallResponse, fn: Callable[..., Any]):
+    metadata: Metadata = fn.__annotations__.get("metadata", {})
```

Comment:
there is a `get_metadata` utility function is base, use that here instead

Reply:
done